android/sdk/src/main/java/org/jitsi/meet/sdk/JitsiMeetOngoingConferenceService.java

```diff
@@ -99,22 +94,18 @@ public static void doLaunch(Context context, HashMap<String, Object> extraData)
         }
     }
 
-    public static Object onRequestPermissionsResult(int requestCode, Context context) {
-        if (permissionsPromises.containsKey(requestCode)) {
-
-            // If request is cancelled, the result arrays are empty.
-            Promise permissionsPromise = permissionsPromises.get(requestCode);
+    private static PermissionListener getNotificationPermissionListener(Context context, HashMap<String, Object> extraData) {
+        return new PermissionListener() {
+            @Override
+            public boolean onRequestPermissionsResult(int i, String[] strings, int[] ints) {
+                if (ContextCompat.checkSelfPermission(context, POST_NOTIFICATIONS) == PackageManager.PERMISSION_GRANTED) {
+                    doLaunch(context, extraData);
 
-            if (ContextCompat.checkSelfPermission(context, POST_NOTIFICATIONS) == PackageManager.PERMISSION_GRANTED) {
-                permissionsPromise.resolve("authorized");
-            } else {
-                permissionsPromise.resolve("denied");
+                    return true;
+                }
+                return false;
```

Comment:
We need to return `true` always, so the listener is removed: https://github.com/facebook/react-native/blob/b0f6a0565057d363c2436f0feb49ab7874216ed9/packages/react-native/ReactAndroid/src/main/java/com/facebook/react/modules/core/PermissionListener.java#L22 We don't want to keep it around.